src/test/scala/examples/Adder.scala

```diff
@@ -45,13 +45,13 @@ class AdderExerciser extends Exerciser {
     val count = 20 // this forces ranges to all be the same size
     Range(start, start + count)
   }
-  val in0_vec = Vec(range(x_range_start).map(UInt(_)))
-  val in1_vec = Vec(range(y_range_start).map(UInt(_)))
+  val in0_vec = Vec(range(x_range_start).map((_).asUInt()))
```

Comment:
I don't think you need the parentheses, you should just be able to write `_.asUInt`